src/app/components/elements/svg/DifficultyIcons.tsx

```diff
@@ -1,16 +1,15 @@
 import React from "react";
 import {Difficulty} from "../../../../IsaacApiTypes";
 import {calculateHexagonProportions, Hexagon} from "./Hexagon";
-import {difficultyLabelMap, difficultyShortLabelMap, siteSpecific} from "../../../services";
+import {difficultyLabelMap, difficultyShortLabelMap} from "../../../services";
 import classnames from "classnames";
 import {Rectangle} from "./Rectangle";
 
 // Difficulty icon proportions
 const difficultyIconWidth = 25;
 const difficultyIconXPadding = 1.5;
 const yPadding = 2;
-const difficultyCategories = ["P", "C"];
-const difficultyCategoryLevels = siteSpecific([1, 2, 3], [1, 2]);
+const difficultyCategoryLevels = [1, 2];
```

Comment:
Did we come to a final decision on discarding some of the difficulty levels? You seem to eliminating some of them here?

Reply:
That's been opened as a separate ticket.  For now I've just removed siteSpecific and left the site functionality unchanged; if it's decided that we want to enable those difficulties for CS I'll change it to [1, 2, 3] as part of the other ticket!